### PR TITLE
Make non-end incremented Ledger derivation path imports possible

### LINKED ISSUE #1105: Update default Ledger derivation path
--> -m/44'/60'/0'/0,

### PR DESCRIPTION
**Pull request by fernya**
Fixes #1105 

Other wallets may increment different parts of the original path.
I chose character 'x' to signal the _only_ place where the root
path should be incremented. If 'x' is not present, we do the
original stuff - add another incrementing part at the end.

Screenshot 2022-03-08 at 10 03 31

### CODE REVIEW

File: ui/components/Onboarding/OnboardingDerivationPathSelect.tsx [hunk 1]
```diff
@@ -11,10 +11,17 @@ import SharedModal from "../Shared/SharedModal"
...
 
 // TODO make this network specific
-const initialDerivationPaths: { value: string; label: string }[] = [
+const initialDerivationPaths: {
+  value: string
+  label: string
+}[] = [
+  {
+    value: "m/44'/60'/x'/0/0",
+    label: "MM Live",
```
Comment (mhluongo) [inline ui/components/Onboarding/OnboardingDerivationPathSelect.tsx:20]: What is "MM Live"?

File: ui/components/Onboarding/OnboardingDerivationPathSelect.tsx [hunk 2]
```diff
@@ -11,10 +11,17 @@ import SharedModal from "../Shared/SharedModal"
...
+  },
   {
     value: "m/44'/60'/0'/0",
-    label: "Ethereum",
+    label: "BIP44",
```
Comment (mhluongo) [inline ui/components/Onboarding/OnboardingDerivationPathSelect.tsx:24]: No one will know that this means I'm afraid.

**Review by mhluongo — COMMENTED** [review]
Quick thoughts!

File: ui/components/Onboarding/OnboardingDerivationPathSelect.tsx (hunk 1 above)
Comment (fernya) [reply ui/components/Onboarding/OnboardingDerivationPathSelect.tsx:20]: When I wrote that, I was unsure if that was just a foxy thing, or Ledger Live does that too.
RN i know that Ledger Live also does that!

File: ui/components/Onboarding/OnboardingDerivationPathSelect.tsx (hunk 2 above)
Comment (fernya) [reply ui/components/Onboarding/OnboardingDerivationPathSelect.tsx:24]: Reverting back, thanks!

**Review by cairomassimo — APPROVED** [review]
Looks ok!

**Pull request merged**